packages/server-wallet/src/__test__/create-and-close-channel/ledger-funding.test.ts

```diff
@@ -476,10 +476,8 @@ describe('Funding multiple channels concurrently (in bulk)', () => {
       allocations: [{allocationItems}],
     } = ledger;
 
-    expect(ledger).toMatchObject({
-      turnNum: 5,
-      status: 'running',
-    });
+    expect(ledger.turnNum === 5 || ledger.turnNum === 7).toBe(true);
+    // there is a race condition which means the turNum should be either 5 or 7
```

Comment:
Would it be better to remove the race condition? Something like:
```ts
const results = [await createMessageAndJoinBatch(), wait createMessageAndJoinBatch()]
```

Reply:
I'm also a bit confused about what race condition would cause different turn numbers here.

Reply:
I think the intent of the test is to prove that the wallet can handle multiple requests happening concurrently

Reply:
> Would it be better to remove the race condition? Something like:

I think that depends on whether we think  concurrent usage of the API (as currently expressed in this test) is a desirable / useful thing to test or support. 

About the race condition: @snario may understand this best?

Reply:
Seems that we should allow the race condition but relax the assertion on the ledger channel turn number. 

As I understand it, the wallet may or may not batch multiple ledger updates into one state update, depending on when it processes request. a54ccdc